frontend/src/routes/+page.svelte

```diff
@@ -1,3 +1,33 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { get } from '$utils/fetch';
+  import { _ } from 'svelte-i18n';
+
+  interface Course {
+    name: string;
+    user: {
+      first_name: string;
+    };
+  }
+  let searchQuery = '';
+  let courses: Course[] = [];
+
+  async function getCourses() {
+    const response = await get('/v1/courses');
+    const data = await response.json();
+    courses = data as Course[];
+  }
+
+  function submitForm() {
+    if (searchQuery) {
+      const query = encodeURIComponent(searchQuery.trim());
+      window.location.href = `/courses?search=${query}`;
```

Comment:
It's better to use [goto](https://kit.svelte.dev/docs/modules#$app-navigation-goto) here to prevent full page reloading.
We've used `window.location.href =` for sign in/up to trigger full page reloading because we need to update full page. Here we don't need it

Reply:
ready